backend/eventd/eventd_test.go

```diff
@@ -69,6 +70,10 @@ func TestEventHandling(t *testing.T) {
 
 	// Make sure the event has been marked with the proper state
 	assert.Equal(t, types.EventPassingState, event.Check.State)
+	fmt.Printf("Event Timestamp: %d\n", event.Timestamp)
+	fmt.Printf("Check's LastOK: %d\n", event.Check.LastOK)
+	fmt.Printf("badEvent Timestamp: %d\n", badEvent.Timestamp)
```

Comment:
@grepory its my understanding that `badEvent` should have an older timestamp but it doesn't. IF you pull down this branch and run `$ go test ./... -run=TestEventHandling -v` you'll see they are all the same. Even with modifying `FixtureCheck` to sleep 5 and print `UnixNano()`. Am I missing some mutations somewhere?

This was just extra code I was writing to check if my test was valid or not.

Reply:
Figured it out!